### ISSUE TITLE
Warning printing to stdout instead of stderr

### ISSUE THREAD

**Issue by spotiris** [tensorflow/tensorflow#29949]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Y
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu Linux 19.04
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.13.0-rc2
- Python version: 3.7
- CUDA/cuDNN version: 7.0
- GPU model and memory: Titan X GTI 12GB

**Describe the current behavior**
When importing from tf.contrib a warning is raised to do with the sunsetting of tf.contrib, as expected.
However unlike every other error in tensorflow, it is written to STDOUT instead of STDERR.
This causes critical problems for applications that require STDOUT to transfer data.

**Describe the expected behavior**
It is expected the warning is written to STDERR.

**Code to reproduce the issue**
```python3
import tensorflow as tf
print(tf.contrib.image.connected_components)
```

**Other info / logs**
The following warning is printed:                                                                                                                                                                         
```
WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.                                                                                                 
For more information, please see:                                                                                                                                              
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md                                                                                        
  * https://github.com/tensorflow/addons                                                                                                                                       
If you depend on functionality not listed there, please file an issue.                 
```

**Comment by achandraa** [comment]
Deprecation warning messages are needed to inform people who want to use an actual version of TensorFlow what is going to happen in the future, and give them time to adapt. However, we can disable these messages for us using private API : 

import tensorflow.python.util.deprecation as deprecation
deprecation._PRINT_DEPRECATION_WARNINGS = False 

Please let us know if that resolves the issue. Thanks!

**Comment by spotiris** [comment]
@achandraa I have gotten around the problem already by using contextlib.redirect_stdout.
Thank you for the better alternative using the deprecations private API.

My application works fine with either of these work-arounds, but is it intentional that this error outputs to STDOUT instead of STDERR?
I think that is quite unusual and unexpected.

**Comment by achandraa** [comment]
As of now, we are providing these warnings just to make user aware of things which are going to happen in the future version and give them time to adapt. You can suppress them using the APIs provided. Hope this answered your query. Thanks!

**Comment by mihaimaruseac** [comment]
This has been fixed in master, there probably was a bug in 1.13-rc0.

```
mihaimaruseac@ankh:/tmp$ mkdir 5
mihaimaruseac@ankh:/tmp$ cd 5
mihaimaruseac@ankh:/tmp/5$ virtualenv .
Running virtualenv with interpreter /usr/bin/python2
New python executable in /tmp/5/bin/python2
Also creating executable in /tmp/5/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
mihaimaruseac@ankh:/tmp/5$ source bin/activate
(5) mihaimaruseac@ankh:/tmp/5$ pip install tf_nightly
... <snip> ...
Successfully installed absl-py-0.7.1 astor-0.8.0 backports.weakref-1.0.post1 enum34-1.1.6 funcsigs-1.0.2 futures-3.2.0 gast-0.2.2 google-pasta-0.1.7 grpcio-1.21.1 h5py-2.9.0 keras-applications-1.0.8 keras-preprocessing-1.1.0 markdown-3.1.1 mock-3.0.5 numpy-1.16.4 protobuf-3.8.0 six-1.12.0 tb-nightly-1.14.0a20190614 termcolor-1.1.0 tf-estimator-nightly-1.14.0.dev2019062701 tf-nightly-1.15.0.dev20190627 werkzeug-0.15.4 wrapt-1.11.2
(5) mihaimaruseac@ankh:/tmp/5$ cat test.py 
import tensorflow as tf
print(tf.contrib.image.connected_components)
(5) mihaimaruseac@ankh:/tmp/5$ python test.py 2> /dev/null
<function connected_components at 0x7fe6b0811410>
```
